dist/pythonlibs/riotctrl_shell/ieee802154.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2019-20 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import re
+
+from riotctrl.shell import ShellInteraction
+
+# ==== ShellInteractions ====
+
+class Config_phy(ShellInteraction):
```

Comment:
All these shell interactions can be one class IMHO. e.g. `IEEE802154Phy(ShellInteraction):`